packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -130,10 +130,11 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <CustomPanel
+    <Drawer
+      isOpen
       title={`Custom View: ${props.customView.defaultLabel}`}
       onClose={props.onClose}
-      size={panelSize}
+      size={panelSize === 'small' ? 10 : 30}
```

Comment:
FYI @CarlosCortizasCT  Since the drawer can only have 3 sizes and "Scale" is not one of them then this is the only option to make sure that the custom view loader is as big as possible if the panel size is not small.